controllers/synapse/synapse_mautrixsignal_service.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package synapse
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	synapsev1alpha1 "github.com/opdev/synapse-operator/apis/synapse/v1alpha1"
+)
+
+// serviceForSynapse returns a synapse Service object
+func (r *SynapseReconciler) serviceForMautrixSignal(s *synapsev1alpha1.Synapse, objectMeta metav1.ObjectMeta) (client.Object, error) {
+	service := &corev1.Service{
+		ObjectMeta: objectMeta,
+		Spec: corev1.ServiceSpec{
+			Ports: []corev1.ServicePort{{
+				Name:       "http",
```

Comment:
Not sure if the port name alters anything. But given the custom port number I assume not. Therefore it could be a more descriptive port name that represents the specific service and not http that represent generally port 80.

Reply:
Although I have the feeling we ran through this before.

Reply:
You're right, we've been through this before. I renamed the port of all services as they were all called "http"

Reply:
Good!